deployer/templates/fluentd.yaml

```diff
@@ -329,13 +323,9 @@ parameters:
   name: USE_MUX
   value: "false"
 -
-  description: 'Configure MUX CLIENT (false|true).'
-  name: USE_MUX_CLIENT
-  value: "false"
--
-  description: 'Configure MUX SERVER (false|true).'
-  name: MUX_ALLOW_EXTERNAL
-  value: "false"
+  description: 'Configure MUX CLIENT MODE (minimal|full_no_k8s_meta).'
```

Comment:
@portante Give me a short, no more than 2 or 3 or 4 very short words, that I can use, that will explain that this is the mode to use for fluentd if you want to do perform all of the processing except the kubernetes metadata enrichment, because 'enrich', 'normalize', etc. do not mean that.